src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -57,7 +57,7 @@ curie_map:
   IEDB: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/IEDB/"
   PMID: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/PMID/"
   KEGG: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/KEGG/"
-  ICD11: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/"
```

Comment:
## Delete old prefix?
- [ ] @matentzn Choose an option

a. Delete `ICD11`, Add `ICD11_FOUNDATION`
b. Keep both
c. Add `ICD11_FOUNDATION` and add the old URI prefix http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/ to it as a synonym in the EPM

I chose 'a' because I think the old prefix map entry was just a placeholder and had no usages. However I was wrong, there is one usage I found, but it is also using a different prefix map.

## Other ICD11 prefix usages
- [ ] @matentzn Review and advise
<details><summary>Details</summary>
<p>

The only usage I found was in `ordo.sssom.tsv`. What should we do about this?

```tsv
# curie_map:
# ...
#   icd11: https://icd.who.int/browse11/l-m/en#/http://id.who.int/icd/entity/
# ...
Orphanet:10	48,XXYY syndrome	oboInOwl:hasDbXref	icd11:LD50.3Y	semapv:UnspecifiedMatching
```

Concens / questions
I guess this is probably only 1 question and (1 - 3) are different options.
1. Retain `icd11` prefix globally in `mondo-ingest`?
I am thinking for compatibility, but I guess with `curies.Converter` and other tooling its not a problem. Is this SSSOM published by Orphanet or us? If us, I'm surprised we have this prefix but it doesn't match the one we had in our `mondo-ingest` config, which was `ICD11: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/`
2. Add this as a prefix synonym?
We now have `ICD11_FOUNDATION: /http://id.who.int/icd/entity/`. Should we add as a synonym to that?
3. If this ORDO SSSOM is something that we maintain, should we update it to our new chosen prefix map? Wouldn't that break things?

</p>
</details>

Reply:
You are right, to avoid clashes with ORDO and DOID, lets always add "foundation" (either `icd11.foundation` which is the bioregistry way, or `ICD11_FOUNDATION` which is the OBO Foundry way - decide how you please)

Reply:
Alright, so sounds like for (1) you're selecting (d) which is a new option and basically the same thing as (2.a). This means we will keep `ICD11` but replace its "unknown_prefix" URI prefix with the one I just found in `ordo.sssom.tsv`. And I will add `icd11.foundation` and the [2 other new ones](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1485632153) we found.
- [ ] @joeflack4 Update `ICD11`
- [ ] @joeflack4 Add [3 new ICD11 foundation prefixes](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1485632153)

Reply:
Done!